ts/parts-3d/Column.ts

```diff
@@ -140,10 +130,10 @@ wrap(H.Series.prototype, 'justifyDataLabel', function (
 seriesTypes.column.prototype.translate3dPoints = function (): void {};
 seriesTypes.column.prototype.translate3dShapes = function (): void {
 
-    var series = this as Highcharts.ColumnSeries,
+    var series: Highcharts.ColumnSeries = this,
         chart = series.chart,
         seriesOptions = series.options,
-        depth = seriesOptions.depth || 25,
+        depth = (seriesOptions as any).depth,
```

Comment:
seriesOptions has own type (`SeriesOptions` or `ColumnSeriesOptions`), use it instead of `any`

Reply:
Unfortunately not possible for now due to a fact that for now depth might be undefined (it is added as an optional parameter in TS). I have also discussed this line with Pawel and we both agreed that before finding more global solution for similar use-cases we should not focus on that issue.